ts/IterableInboxMessageDisplay.tsx

```diff
@@ -0,0 +1,61 @@
+'use strict'
+import React, { Component } from 'react'
+import { Text, View, StyleSheet } from 'react-native'
+import Icon from 'react-native-vector-icons/Ionicons'
+import Message from "./messageType"
+
+type MessageDisplayProps = {
+   message: Message,
+   returnToInbox: Function    
+}
+
+const IterableInboxMessageDisplay = ({ message, returnToInbox }: MessageDisplayProps) => {
+   const messageTitle = message.inboxMetadata.title
+
+   return(
+      <View> 
+         <View style={styles.returnButtonContainer}>
+            <Icon 
+               name="ios-arrow-back" 
+               style={styles.returnButton} 
+               onPress={() => {returnToInbox()}}/>
```

Comment:
`onPress={() => returnToInbox()}/>` makes this slightly more readable and causes no issues.

Reply:
I will make the edit thanks.